pkg/controller/apimanager/apimanager_controller_test.go

```diff
@@ -262,43 +188,7 @@ func TestAPIManagerControllerUpgrade(t *testing.T) {
 	}
 
 	if !res.Requeue {
-		t.Error("upgrade not expected to finish. Should requeue before update system-app pre-hook pod")
-	}
-
-	res, err = r.Reconcile(req)
-	if err != nil {
-		t.Fatalf("reconcile: (%v)", err)
-	}
-
-	if !res.Requeue {
-		t.Error("upgrade not expected to finish. Should requeue after migrating smtp ConfigMap to system-smtp Secret")
-	}
-
-	res, err = r.Reconcile(req)
-	if err != nil {
-		t.Fatalf("reconcile: (%v)", err)
-	}
-
-	if !res.Requeue {
-		t.Error("upgrade not expected to finish. Should requeue after updating smtp envvars in system-sidekiq")
-	}
-
-	res, err = r.Reconcile(req)
-	if err != nil {
-		t.Fatalf("reconcile: (%v)", err)
-	}
-
-	if !res.Requeue {
-		t.Error("upgrade not expected to finish. Should requeue after updating smtp envvars in system-app")
-	}
-
-	res, err = r.Reconcile(req)
-	if err != nil {
-		t.Fatalf("reconcile: (%v)", err)
-	}
-
-	if !res.Requeue {
-		t.Error("upgrade not expected to finish. Should requeue before update annotations")
+		t.Error("upgrade procedure tries to reques on finished")
```

Comment:
Does this have a typo? What do we want to say here?

Reply:
typo. 

Upgrade procedure ends up the work asking for a requeue